app/models/v2/value_definition.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Stores information about value definitions. This comes from SCAP.
+module V2
+  # Model for Value Definitions
+  class ValueDefinition < ApplicationRecord
+    belongs_to :security_guide, class_name: 'V2::SecurityGuide'
+
+    sortable_by :title
+
+    scoped_search on: :title, only_explicit: true, operators: %i[like unlike eq ne in notin]
```

Comment:
I am not sure about the operators here, we would need the plans from @xmicha82 😞